lib/upgrade_format_task.rb

```diff
@@ -52,12 +58,10 @@ def upgrade(store, name, options = {})
 
     if options[:all]
       if migrations_done > 0
-        Machinery::Ui.puts "Upgraded #{migrations_done} system descriptions successfully."
+        Machinery::Ui.puts "Upgraded #{migrations_done} #{"system description".pluralize(migrations_done)}."
```

Comment:
Line is too long. [108/100]<br>Prefer single-quoted strings inside interpolations.